modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,39 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta Processor
+
+:FeatureName: The Cumulative to Delta Processor
+include::snippets/technology-preview.adoc[]
+
+This processor converts monotonic, cumulative-sum, and histogram metrics to monotonic delta metrics.
+
+[NOTE]
+====
+This processor does not convert non-monotonic sums and exponential histograms.
+====
+
+.Example of an OpenTelemetry Collector custom resource with an enabled Cumulative to Delta Processor
+[source,yaml]
+----
+config: |
+  processors:
+    cumulativetodelta:
+      include: # <1>
+        match_type: strict
+        metrics:
+        - first_metric
+        - second_metric
```

Comment:
```suggestion
        - <first_metric>
        - <second_metric>
```

Reply:
User-replaced variables should be in brackets `<>`. See [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values) (though in the OCP docs, we don't italicize) and the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) ([here too](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values)).

Reply:
Thank you. Fixed.